collections/_article/upcoming-serious-web-performance-boost.md

```diff
@@ -0,0 +1,50 @@
+
+And it blew our minds.
+
+## Benchmarks
+
+Our resident benchmark expert Hugo Locurcio (better known as [Calinou](https://github.com/Calinou/)) did run the numbers for us on a stress test I made.
+
+<div class="note" markdown=1>
+**Note:** You may try to replicate his results, but be aware that he has a beast of a machine. Here are his PC's specifications:
+
+- **CPU:** Intel Core i9-13900K
+- **GPU:** NVIDIA GeForce RTX 4090
+- **RAM:** 64 GB (2×32 GB DDR5-5800 C30)
+- **SSD:** Solidigm P44 Pro 2 TB
+- **OS:** Linux (Fedora 42)
+</div>
+
+I built a Jolt physics stress test from a scene initially made by [passivestar](https://bsky.app/profile/passivestar.bsky.social). By spawning more and more barrels into the contraption, we can easily test the performance difference between the WASM SIMD build and the other.
+
+| | Without WASM SIMD | With WASM SIMD | Improvement (approx.) |
+| :---: | :---: | :---: | :---: |
+| Test links | <a href="https://adamscott.github.io/godot-physics-demo-without-simd/" target="_blank">Link</a> | <a href="https://adamscott.github.io/godot-physics-demo-with-simd/" target="_blank">Link</a> | - |
+| Firefox 138<br>("+100 barrels" 3 times) | ![Firefox 138 "+ 100 barrels" 3 times without SIMD](/storage/blog/upcoming-web-serious-performance-boost/calinou-firefox138-3times-without-wasm-simd.webp) | ![Firefox 138 "+ 100 barrels" 6 times without SIMD](/storage/blog/upcoming-web-serious-performance-boost/calinou-firefox138-3times-with-wasm-simd.webp) | **2×** |
+| Firefox 138<br>("+100 barrels" 6 times) | ![Firefox 138 "+ 100 barrels" 3 times without SIMD](/storage/blog/upcoming-web-serious-performance-boost/calinou-firefox138-6times-without-wasm-simd.webp) | ![Firefox 138 "+ 100 barrels" 6 times without SIMD](/storage/blog/upcoming-web-serious-performance-boost/calinou-firefox138-6times-with-wasm-simd.webp) | **10.17×** |
+| Chromium 134<br>("+100 barrels" 3 times) | ![Chromium 134 "+ 100 barrels" 3 times without SIMD](/storage/blog/upcoming-web-serious-performance-boost/calinou-chromium134-3times-without-wasm-simd.webp) | ![Chromium 134 "+ 100 barrels" 6 times without SIMD](/storage/blog/upcoming-web-serious-performance-boost/calinou-chromium134-3times-with-wasm-simd.webp) | **1.37×** |
+| Chromium 134<br>("+100 barrels" 6 times) | ![Chromium 134 "+ 100 barrels" 3 times without SIMD](/storage/blog/upcoming-web-serious-performance-boost/calinou-chromium134-6times-without-wasm-simd.webp) | ![Chromium 134 "+ 100 barrels" 6 times without SIMD](/storage/blog/upcoming-web-serious-performance-boost/calinou-chromium134-6times-with-wasm-simd.webp) | **14.17×** |
+
+## What it means for your games
+
+Starting with 4.5 dev 5, you can expect your games to run a little bit more smoothly, without having to do anything. Especially when things get chaotic (for your CPU). It isn't a silver bullet for poorly optimized games, but it will help. And it will not help for GPU rendering bottlenecks.
```

Comment:
```suggestion
Starting with 4.5 dev 5, you can expect your web games to run a little bit more smoothly, without having to do anything. Especially when things get chaotic (for your CPU). It isn't a silver bullet for poorly optimized games, but it will help. And it will not help for GPU rendering bottlenecks.
```